pkg/cluster/cluster_test.go

```diff
@@ -59,18 +61,35 @@ func TestClusterGetMissing(t *testing.T) {
 
 func TestClusterApplyKIND(t *testing.T) {
 	f := newFixture(t)
+	f.dmachine.os = "darwin"
 
+	assert.Equal(t, false, f.d4m.started)
 	kindAdmin := newFakeAdmin(f.config)
 	f.controller.admins[ProductKIND] = kindAdmin
 
 	result, err := f.controller.Apply(context.Background(), &ctlptlapi.Cluster{
 		Product: string(ProductKIND),
 	})
 	assert.NoError(t, err)
+	assert.Equal(t, true, f.d4m.started)
 	assert.Equal(t, "kind-kind", kindAdmin.created.Name)
 	assert.Equal(t, "kind-kind", result.Name)
 }
 
+func TestClusterApplyDockerForMacNoCluster(t *testing.T) {
```

Comment:
This is testing whether we restart d4m if both the cluster hasn't started and we need to adjust CPU, right? might also be worth testing:
1. cluster needs start but cpu doesn't need adjusting
2. cpu needs adjusting but cluster doesn't need starting
3. no restart needed

Reply:
done